### DISCUSSION TITLE
Improve search in Apple CarPlay

### DISCUSSION THREAD

**Discussion by br4yd** [osmandapp/OsmAnd#16420] [Ideas]
The app and it's customizability is awesome! However it's really hard to search for places in Apple CarPlay. Let's say I use the app on the phone I can find and navigate to a restaurant by searching the restaurants name. This does not work in Apple CarPlay. I have to explicitly search for the address.

Which makes it more difficult is that I in addition can somehow not search for address + house number in Apple CarPlay. I can only navigate to the street itself but I'm never able to enter the specific house number.

**Comment by vshcherb** [comment]
Yes we have issues that search is not complete in Car Play we are preparing a new release that coordinates, poi and address search will work correctly as it works on the phone itself
